gapic/src/main/com/google/gapid/util/Paths.java

```diff
@@ -400,6 +413,64 @@ public static int compare(Path.Command a, Path.Command b) {
     }
   }
 
+  public static Path.Any reparent(Path.Any path, Path.State newState) {
```

Comment:
There must be a cleaner / less fragile way of getting / setting parents. Unfortunately I don't know what that is right now.